### ISSUE TITLE
Wrong marker frequency in measurement mode

### ISSUE THREAD

**Issue by pedropedro** [OpenHantek/OpenHantek6022#62]
Hi Ho-Ro,
please see attached screenshot below - Probe connected to calibration out, frequency set to 20kHz.
"Measure" button activated. Markers set at the borders of a single signal period, i.e. 50µs.
Everithing (except the frequency in the right upper corner) is calculated / displayed correctly,
see green boxes in the screenshot. **The red boxed marker frequency is IMHO wrong - 5kHz.**
Shouldn't it be 20 kHz ?

If I move both markers to get measured time period for instance of   xy.z µs , the *marker*
frequency follows the markers *allways* showing x.yz kHz  .... 45.6µs yields 4.56 kHz ...
basically time [µs] / 10.00 --> freq [kHz]

**Computer environment:**
 - OS: Xubuntu 19.09 amd64 running on VirtualBox 6.1.4 with Guest Additions 6.1.4

**Scope device:**
 - Hantek 6022BE
 - Program top line: OpenHantek6022 3.0.2 - FW 0204
 - Input signal: calibration out 20 kHz
 - Probe setting: probe X1

![OH6022-3 0 2-FW0204](https://user-images.githubusercontent.com/2650935/76686484-08254880-661c-11ea-981a-0505524e17c4.png)

MfG Pedro

**Comment by pedropedro** [comment]
Okay, my mistake, those 5 kHz shown is the distance in [kHz] at the lowest axis, where the marker labels ("1" and "2") are placed.

MfG Pedro
